src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import java.util.Date;
+
+public class RandomDataProvider {
```

Comment:
this is not `dataprovider` it doesn't have sense to have additional class which is wrapper for java util function.
Please use `DateTime.now().getMillis()` from `org.joda.time.DateTime`